pywbem_mock/_defaultinstanceprovider.py

```diff
@@ -373,8 +378,16 @@ class InstanceWriteProvider(BaseProvider):
 
     Note that user-defined providers may, in turn, call the default providers
     in this class.
+
+    Attributes:
+
+      provider_type (:term:`string`):
+        Keyword defining the type of request the provider will service.
+        The type for this class is predefined as 'instance'
```

Comment:
This way of describing the `provider_type` attribute causes Sphinx not to create proper links to the attribute. I verified that when changing this into line comments, it works fine:

```
    #:  provider_type (:term:`string`):
    #:    Keyword defining the type of request the provider will service.
    #:    The type for this class is predefined as 'instance'
    provider_type = 'instance'
```

Reply:
DONE